content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -498,6 +538,310 @@ spec:
 {{% /tab %}}
 {{< /tabs >}}
 
+## Webhook request and response
+
+### Request
+
+Webhooks are sent a POST request, with `Content-Type: application/json`,
+with a `ConversionReview` API object in the `apiextensions.k8s.io` API group
+serialized to JSON as the body.
+
+Webhooks can specify what versions of `ConversionReview` objects they accept
+with the `conversionReviewVersions` field in their CustomResourceDefinition:
+
+{{< tabs name="conversionReviewVersions" >}}
+{{% tab name="apiextensions.k8s.io/v1" %}}
+```yaml
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+...
+spec:
+  ...
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1", "v1beta1"]
+      ...
+```
+
+`conversionReviewVersions` is a required field when creating 
+`apiextensions.k8s.io/v1` custom resource definitions.
+Webhooks are required to support at least one `ConversionReview`
+version understood by the current and previous API server.
+{{% /tab %}}
+{{% tab name="apiextensions.k8s.io/v1beta1" %}}
+```yaml
+# Deprecated in v1.16 in favor of apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+...
+spec:
+  ...
+  conversion:
+    strategy: Webhook
+    conversionReviewVersions: ["v1", "v1beta1"]
+    ...
+```
+
+If no `conversionReviewVersions` are specified, the default when creating 
+`apiextensions.k8s.io/v1beta1` custom resource definitions is `v1beta1`.
+{{% /tab %}}
+{{< /tabs >}}
+
+API servers send the first `ConversionReview` version in the `conversionReviewVersions` list they support.
+If none of the versions in the list are supported by the API server, the custom resource definition will not be allowed to be created.
+If an API server encounters a conversion webhook configuration that was previously created and does not support any of the `ConversionReview`
+versions the API server knows how to send, attempts to call to the webhook will fail.
+
+This example shows the data contained in an `ConversionReview` object
+for a request to convert `CronTab` objects to `example.com/v1`:
+
+
+{{< tabs name="ConversionReview_request" >}}
+{{% tab name="apiextensions.k8s.io/v1" %}}
+```yaml
+{
+  "apiVersion": "apiextensions.k8s.io/v1",
+  "kind": "ConversionReview",
+  "request": {
+    # Random uid uniquely identifying this conversion call
```

Comment:
nit: comments not supported in json. Can you inline comments in json embedded in yaml?